core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.spec.ts

```diff
@@ -255,13 +255,17 @@ describe('DotPagesFavoritePanelComponent', () => {
         it('should set panel inputs and attributes', () => {
             const elem = de.query(By.css('p-panel'));
             expect(elem.nativeElement.classList.contains('dot-pages-panel__expanded')).toBeFalse();
-            expect(elem.componentInstance['iconPos']).toBe('start');
+            expect(elem.componentInstance['iconPos']).toBe('end');
             expect(elem.componentInstance['expandIcon']).toBe('pi pi-angle-down');
             expect(elem.componentInstance['collapseIcon']).toBe('pi pi-angle-up');
-            expect(elem.componentInstance['header']).toBe('favoritePage.panel.header');
             expect(elem.componentInstance['toggleable']).toBe(true);
         });
 
+        it('should have a star icon in the header', () => {
+            const elem = de.query(By.css('.dot-pages-panel__header'));
+            expect(elem.nativeElement.querySelector('.pi-star-fill')).toBeTruthy();
```

Comment:
add a `data-testId` to the icon and query it using that, using style classes can lead to more work in the future if we want to change that class or that icon